incubator/hnc/hack/test-issue-716.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
This test seems to do a _lot_ of stuff. Can we break it up into multiple smaller and simpler tests? E.g.:

1. Create a->b where b is a subns. Try to delete a (fail). Try to delete b (succeed). Try to delete a (succeed).
2. As in #1 but b is a full namespace.
3. Create a->b->c where b and c are subnamespaces. Try to delete b; fail. Try to delete c; succeed. Try to delete b: succeed.

Reply:
SG. Split into 3 bash files.